apps/api-gateway/src/main.ts

```
@@ -88,15 +88,64 @@ async function bootstrap(): Promise<void> {
-  } catch (err) {
-    Logger.warn('Skipping EcosystemSwaggerFilter due to error', err as Error);
+ let document = SwaggerModule.createDocument(app, options);
+try {
+  const ecosystemFilter = app.get(EcosystemSwaggerFilter);
+  document = await ecosystemFilter.filterDocument(document);
+} catch (err) {
+  Logger.warn('Skipping EcosystemSwaggerFilter due to error', err as Error);
+}
+
+// 🔹 Helper to filter APIs by tag
+function filterByTags(doc: any, tagNames: string[]) {
```

ESLint

[error] Missing return type on function.

(`@typescript-eslint/explicit-function-return-type`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.

(guard-for-in)